### DISCUSSION TITLE
how to make Bridge not capable

### DISCUSSION THREAD

**Discussion by superPN** [lldpd/lldpd#643] [Q&A]
Hi,
I don't know what this bridge represents and what should I do to make it "not capable"
![微信图片_20240319164954](https://github.com/lldpd/lldpd/assets/114060304/ecebd17e-a116-458c-a530-8c9dd3021280)

My config:
Ubuntu20.04
I downloaded lldpd using "sudo apt-get install lldpd"
lldpd -v    1.0.4

Thanx for your work

**Comment by vincentbernat** [comment]
Not possible. Any Linux can act as a bridge, so it is hardcoded.
